changelog/29475.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: Replaces instances of custom css class with Hds::Badge component
```

Comment:
Nice initiative adding a changelog! However, since this is mostly an internal change (with near-negligible user facing changes) we don't need to include a changelog for this PR. Users read changelogs to infer how an update may affect them, so if this HDS component swap was a major accessibility improvement, then it'd be useful to include. PRs without changelogs can use the `pr/no-changelog` label to skip the github action. (But that action is not required to pass to merge PRs)